### DISCUSSION TITLE
there are an error alarm when I want to add a new folder

### DISCUSSION THREAD

**Discussion by TobinYang** [requarks/wiki#4846] [Error / Bug Report]
It will have an error alarm when I want to add a new folder. This page is in the Markdown editor.
<img width="1440" alt="bug" src="https://user-images.githubusercontent.com/43194209/147797567-a23b5c0a-1111-459a-88f9-e5b5d31e4661.png">

**Comment by NGPixel** [comment]
What version of MySQL/MariaDB are you using?

**Reply by TobinYang** [reply]
5.7.31

**Reply by NGPixel** [reply]
https://docs.requarks.io/install/requirements/mysql5

You're using an unsupported version.

**Reply by TobinYang** [reply]
thank you so much
